SPECS/iptables/iptables.spec

```diff
@@ -43,21 +45,22 @@ It contains the libraries and header files to create applications.
     --enable-devel
 
 make V=0
+
 %install
 [ %{buildroot} != "/"] && rm -rf %{buildroot}/*
 make DESTDIR=%{buildroot} install
 ln -sfv ../../sbin/xtables-multi %{buildroot}%{_libdir}/iptables-xml
 #   Install daemon scripts
 install -vdm755 %{buildroot}%{_unitdir}
 install -m 644 %{SOURCE1} %{buildroot}%{_unitdir}
-install -vdm755 %{buildroot}/etc/systemd/scripts
-install -m 755 %{SOURCE2} %{buildroot}/etc/systemd/scripts
-install -m 755 %{SOURCE3} %{buildroot}/etc/systemd/scripts
-install -m 644 %{SOURCE4} %{buildroot}/etc/systemd/scripts
-install -m 644 %{SOURCE5} %{buildroot}/etc/systemd/scripts
+install -vdm755 %{buildroot}%{_sysconfdir}/systemd/scripts
+install -m 755 %{SOURCE2} %{buildroot}%{_sysconfdir}/systemd/scripts
+install -m 755 %{SOURCE3} %{buildroot}%{_sysconfdir}/systemd/scripts
+install -m 644 %{SOURCE4} %{buildroot}%{_sysconfdir}/systemd/scripts
+install -m 644 %{SOURCE5} %{buildroot}%{_sysconfdir}/systemd/scripts
 
 find %{buildroot} -name '*.a'  -delete
-find %{buildroot} -name '*.la' -delete
+find %{buildroot} -type f -name "*.la" -delete -print
```

Comment:
This should use single quotes -- otherwise, the shell will expand the wildcard before passing the arguments to `find(1)`.

Reply:
Looks like this is caused by the linter- will go ahead and make a fix there.